roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -153,6 +153,10 @@ resource "openstack_compute_instance_v2" "cluster_server" {
     azimuth_user_id = "${data.openstack_identity_auth_scope_v3.scope.user_id}"
     azimuth_project_name = "${data.openstack_identity_auth_scope_v3.scope.project_name}"
     azimuth_project_id = "${data.openstack_identity_auth_scope_v3.scope.project_id}"
+    azimuth_workstation_username = "{{ cluster_user_username | default('azimuth') }}"
+    azimuth_workstation_uid = "{{ cluster_user_uid | default('1005') }}"
+    azimuth_workstation_gid = "{{ cluster_user_gid | default('1005') }}"
+    azimuth_workstation_is_sudo = "{{ cluster_user_is_sudo | default(true) }}"
```

Comment:
I am tempted to make this "True" so its going to be like what gets passed from Azimuth into the string value here, but otherwise seems OK.